experimental/gittuf/policy_test.go

```diff
@@ -114,3 +114,18 @@ func TestPullPolicy(t *testing.T) {
 		assert.ErrorIs(t, err, ErrPullingPolicy)
 	})
 }
+
+func TestApplyPolicy(t *testing.T) {
+	localRepo := createTestRepositoryWithPolicy(t, "")
```

Comment:
```suggestion
	repo := createTestRepositoryWithPolicy(t, "")
```

local, remote prefix makes sense for the synchronization tests.